py/desimeter/posparams/fithandler.py

```diff
@@ -50,8 +50,8 @@ def run_best_fits(posid, path, period_days, data_window, savedir,
         return f'{posid}: dropped from analysis (invalid table)'
     _make_petal_xy(table, printf=printf)
     _apply_filters(table, printf=printf)
-    if not table:
-        return f'{posid}: dropped from analysis (no data remaining after filters applied)'
+    if not table or len(table) < 10:
```

Comment:
Should the value here be `data_window`, rather than hard-coded 10?

Reply:
10 was admittedly pulled from a hat, but it was motivated by a case of having only 1 data point passing filters, which clearly was insufficient for fitting 6-8 parameters.  I don't think I want it as wide as the full `data_window`, since I was also motivated by processing a specific calibration run without having to tune the `data_window` option (i.e. if the entire dataset is smaller than the default data window, print a warning but proceed anyway, but don't fiddle with specifying a window of 24 and then find out that there were actually 25 moves that got split into two groups or 23 moves that it would then refuse to process etc.).

Does that seem reasonable (or at least acceptable)?

Reply:
I think that makes sense. Maybe just add an optional argument `min_window=10` to `run_best_fits`. We'll probably never mess with it again, but gives some runtime control without changing code. But I wouldn't bother adding it to the command line tool at this point.

Reply:
Good idea.  I added `min_window=10` option to `run_best_fits`, but I didn't go as far as making it a `fit_posparams` command line option.  We can add that if we find that we need to use that flexibility.